PyISY/Nodes/__init__.py

```diff
@@ -187,8 +187,22 @@ def parse(self, xml):
                                 dimmable = '%' in \
                                     nprop[0].attributes['uom'].value
                                 nval = int(nval.replace(' ', '0'))
+                            # Get the device notes, currently only the Spoken property is saved.
+                            notes_xml = self.parent.conn.getNodeNotes(nid)
```

Comment:
I don't really like the idea of doing this while initializing everything. PyISY already takes a while to start up because it has to grab and parse so much data from the ISY. I would rather have this data fetched on the fly rather than when setting up the Node.